src/SamplesApp/SamplesApp.Shared/App.xaml.cs

```diff
@@ -431,7 +431,7 @@ public static void ConfigureFilters()
 				builder.AddConsole();
 #endif
 
-
+				builder.AddDebug();
```

Comment:
This should not be needed.

Reply:
It was necessary.
So I'm doing something wrong about the Log. Can you take a look?

Reply:
Removed.